go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/cosmos/ledger-cosmos-go v0.11.1
 	github.com/desertbit/timer v0.0.0-20180107155436-c41aec40b27f // indirect
 	github.com/enigmampc/btcutil v1.0.3-0.20200723161021-e2fb6adb2a25
+	github.com/ethereum/go-ethereum v1.10.3
```

Comment:
How do we feel about having a dependency on geth? Are there any issues with that?

Reply:
it's a huge dependency, would be good to avoid it if possible.

Reply:
(right now, it's already brought in by Rosetta though)